revenuecat_examples/MagicWeather/lib/src/views/weather.dart

```diff
@@ -70,7 +72,7 @@ class _WeatherScreenState extends State<WeatherScreen> {
             return StatefulBuilder(
                 builder: (BuildContext context, StateSetter setModalState) {
               return Paywall(
-                offering: offerings.current,
+                offering: offerings!.current!,
```

Comment:
Hmm is Dart not smart enough to understand that this can't be null in the `else`? :(

Reply:
I don't think so... Only Kotlin haha